.gitignore

```diff
@@ -61,7 +61,7 @@ dkms.conf
 *.user
 
 # CMake build directory
-/build
+build/
```

Comment:
what is our official cmake directory name? I usually use cmake. I have also seen walkthru guide also use cmake.

Reply:
Our recommendation for this repo is `build`. `\cmake` already exists and contains cmake scripts.

I think I saw it named `cmake` in C-SDK documentation but for simplicity, we should avoid changing it from the VSCode default.